hack/make-release.sh

```diff
@@ -43,7 +43,7 @@ fi
 # create git release / push to new branch
 git checkout -b "release/${TAG}"
 git add "${SCRIPTDIR}/../release/"
-git add "${SCRIPTDIR}/../kustomize/"
+git add "${SCRIPTDIR}/../kustomize/base/"
```

Comment:
When we run `/hack/make-release.sh`, the files that need to be committed are:
* `/release/kubernetes-manifests.yaml`
* `/release/istio-manifests.yaml`
* `/kustomize/base/adservice.yaml`
* `/kustomize/base/cartservice.yaml`
* etc.